### ISSUE TITLE
How to get the correct range for all parameters

### ISSUE THREAD

**Issue by xxlliu** [cagrikymk/JAX-ReaxFF#18]
Dear author,
Hello! I'm confused about optimizing the parameter range, and I'm not sure if my parameter range is the right one. I would like to ask how I can get the correct range of all parameters.
Thank you for your time！

**Comment by cagrikymk** [comment]
Hello,
You can look at the example dataset files and examine the ranges given in the "params" files.
For example in line "2 1 1 0.1 1.7 2.6", first 3 values determine the parameter index: 2. group, 1. line, 1. value.
The fourth value is the parameter sensitivity which is ignored by JAX-ReaxFF as we rely on gradients to determine that.
The last 2 values determine the parameter range. These values change based on the atom types involved in the corresponding term in the functional form. 

You can create your "params" file by examining these files. Unfortunately, I am not fully familiar with the widest ranges you can choose based on the functional form of the ReaxFF potential. I am planning to contact the developer of ReaxFF (Dr. Adri van Duin) to get more information about the parameters and some guiding ranges for each.

**Comment by xxlliu** [comment]
Thank you for your answer and look forward to your follow-up progress.

**Comment by xxlliu** [comment]
I'm sorry to bother you again.
I want to use jax-reaxff to get the error report of untrained raw force field parameters to the training set. How should I write parameter files?
When I leave the parameter file blank, the program will report an error. I also tried to write the parameter file as "2 1 1 0.1 1.7 1.7" (the corresponding parameter in the force field file is 1.7). The results this way also seem to be problematic, as I used single and double precision programs to get inconsistent and wildly different results.

**Comment by xxlliu** [comment]
Dear author,
Hello!  I'm so sorry to bother you again. Do you have any progress on this issue?
Thank you for your time!
